metron-platform/metron-common/src/main/java/org/apache/metron/common/dsl/functions/resolver/ClasspathFunctionResolver.java

```diff
@@ -235,8 +235,16 @@ public void initialize(Context context) {
     }
 
     FilterBuilder filterBuilder = new FilterBuilder();
-    excludes.forEach(excl -> filterBuilder.exclude(excl));
-    includes.forEach(incl -> filterBuilder.include(incl));
+    excludes.forEach(excl -> {
```

Comment:
This is an intermittent test bug that I noticed and fixed in order to get the tests to run consistently.